### PR TITLE
Doc/4532 faq retain original paths

### LINKED ISSUE #4532: borg backup and btrfs
This is Fedora 30 Beta and borg is version 1.1.9

When I do a backup of a `btrfs` file system I first do a readonly snapshot, then back up the readonly snapshot.

Example
```
btrfs sub snap -r /home /.snapshots/home
borg create -v -stats /mnt/borg_repos::home.20190428 /.snapshots/home
btrfs sub del /.snapshots/home
```
The problem with this approach is that instead of paths starting with `/home` I have now `/.snapshots/home`.

Anything, I can do here.

### PR DESCRIPTION
**Pull request by fantasya-pbem**
fixes #4532

### CODE REVIEW

File: docs/faq.rst [hunk 1]
```diff
@@ -702,6 +702,23 @@ Now Borg will be bandwidth limited. Nice thing about pv is that you can change r
...
 
+How can I avoid unwanted base directories getting stored into archives?
+-----------------------------------------------------------------------
+
+Possible use cases:
+
+- Another file system is mounted and you want to backup it with original paths.
+- You have created a BTRFS snapshot in a ``/.snapshots`` directory for backup.
+
+To achieve this, run ``borg_create`` within the mountpoint/snapshot directory:
```
Comment (ThomasWaldmann) [inline docs/faq.rst]: Hmm, how is this rendered? Is this a link to the borg create docs?

Or should it rather be ```borg create``` (with blank, not underscore)?

File: docs/faq.rst [hunk 2]
```diff
@@ -702,6 +702,23 @@ Now Borg will be bandwidth limited. Nice thing about pv is that you can change r
...
+
+::
+
+    # Example: Some file system mounted in /mnt/rootfs.
+    cd /mnt/rootfs
+    borg create /path/to/repo::rootfs_backup .
```
Comment (ThomasWaldmann) [inline docs/faq.rst]: yeah, even better. we do not need to show the mount.

File: docs/faq.rst (hunk 1 above)
Comment (fantasya-pbem) [reply docs/faq.rst]: That should be a space, not underscore. Fixed.

**Comment by ThomasWaldmann** [comment]
same for 1.1-maint?

**Pull request merged**